segmentation/utils/Spatial_Analysis.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+import math
+import skimage.measure
+from scipy.spatial.distance import cdist
+
+
+def calc_dist_matrix(label_map):
+    """Generate matrix of distances between center of pairs of cells
+        Args
+            label_map: numpy array with unique cells given unique pixel labels
+        Returns
+            dist_matrix: cells x cells matrix with the euclidian
+            distance between centers of corresponding cells"""
+
+    if len(np.unique(label_map)) < 3:
+        raise ValueError("Array must be provided in labeled format")
+
+    props = skimage.measure.regionprops(label_map)
+    a = [None] * len(props)
```

Comment:
It would be more pythonic to create this with a [list comprehension](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions) instead of creating a list full of Nones.